Board.java

```diff
@@ -41,10 +41,12 @@ public boolean checkForWin(char player) {
             int count = 0;
             for (int i=0; i < formation.length; i++) {
                if (state.charAt(formation[i]) == player) count += 1;
-               if (count == 3) return true;
+               if (count == 3) {
+                   return true;
```

Comment:
Either use curly brackets or inline, but not both e.g. few lines above you've inlined.